### PR TITLE
Remove duplicates from list of commits

### LINKED ISSUE #13: Default commits selection
It seems that by default this tool selects the same commit for both left and right which is pointless and leads to a pretty nasty issue - the difftool won't start. It may be a good idea to select last or previous commit for "before"

### LINKED ISSUE #14: Remove duplicates from list of versions
For some reason gitlab may create two version with the same HEAD but different Base/Start SHA pairs after a merge commit occured during work on a merge request (from target branch to a source branch). It does not make sense for our tool to show both versions in a version list because we use only HEAD from them.
Example:

 
--
id | 1503
head_commit_sha | "b2115fe4f350e2568515888b202bf5096e46466b"
base_commit_sha | "6b9121f0326bce5730379553b41b9405aa253091"
start_commit_sha | "6b9121f0326bce5730379553b41b9405aa253091"
created_at | "2019-07-03T20:11:52.108Z"
merge_request_id | 507
state | "collected"
real_size | "1"
1 |  
id | 1502
head_commit_sha | "84815eb18137bade6586016a0a67d727636afa1a"
base_commit_sha | "a42836b26159264b2fdbc3b675eee41c2cfa0b47"
start_commit_sha | "6b9121f0326bce5730379553b41b9405aa253091"
created_at | "2019-07-03T20:09:28.889Z"
merge_request_id | 507
state | "collected"
real_size | "1"
2 |  
id | 1501
head_commit_sha | "84815eb18137bade6586016a0a67d727636afa1a"
base_commit_sha | "a42836b26159264b2fdbc3b675eee41c2cfa0b47"
start_commit_sha | "6b9121f0326bce5730379553b41b9405aa253091"
created_at | "2019-07-03T20:09:07.619Z"
merge_request_id | 507
state | "collected"
real_size | "1"
image

Proposal: remove duplicate rows from Versions list.

### PR DESCRIPTION
**Pull request by GrimMaple**
Fixes #14

I also removed the latest commit from "right" section as having it there is pointless. It fixes #13.

### CODE REVIEW

File: mrHelper/mrHelperForm.cs [hunk 1]
```diff
@@ -825,10 +826,15 @@ private void onMergeRequestSelected()
          comboBoxLeftVersion.Items.Clear();
          comboBoxRightVersion.Items.Clear();
 
-         foreach (var version in getVersions())
+         var versions = getVersions();
+         comboBoxLeftVersion.Items.Add(new VersionComboBoxItem(versions[0]));
```
Comment (denis-adamchuk) [inline mrHelper/mrHelperForm.cs]: can you please resolve #22 at once? it is right here.

File: mrHelper/mrHelperForm.cs [hunk 2]
```diff
@@ -1052,6 +1058,12 @@ public VersionComboBoxItem(string sha, string text, DateTime? timeStamp)
             Text = text;
             TimeStamp = timeStamp;
```
Comment (denis-adamchuk) [inline mrHelper/mrHelperForm.cs]: do we still need old constructor?

File: mrHelper/mrHelperForm.cs (hunk 1 above)
Comment (GrimMaple) [reply mrHelper/mrHelperForm.cs]: Done, sorta

File: mrHelper/mrHelperForm.cs (hunk 2 above)
Comment (GrimMaple) [reply mrHelper/mrHelperForm.cs]: Yup

**Pull request merged**